.github/ISSUE_TEMPLATE/are-you-lost-.md

```diff
@@ -1,7 +1,6 @@
 ---
-name: Are you lost?
-about: If you did not find a bug and do not want to suggest a new feature, you probably
-  have some questions.
+name: Ask a quetion
+about: ... if you need some explanations TODO
```

Comment:
I would not propose "explanations". This is not support forum. We should be aware to split this. So I would prefer a text like:

"If you have an idea or an issue, that does not fit in the other mentioned....".